client/app/private-database/database/archive/dump/private-database-database-archive-dump.html

```diff
@@ -1,16 +1,13 @@
 <div data-ng-controller="PrivateDatabaseArchiveDumpCtrl as archiveDumpCtrl">
     <div data-ovh-alert="{{alerts.main}}"></div>
 
-    <button class="oui-button oui-button_link oui-button_icon-left" type="button"
-            data-ng-click="goToArchivesList()">
-        <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span>
-        <span data-i18n-static="common_back"></span>
-    </button>
-
-    <h2 data-ng-bind-html="tr('privateDatabase_tabs_dumps_change') + ' ' + bdd.name"></h2>
+    <oui-back-button
+        data-on-click="goToArchivesList()"
```

Comment:
You should replace it with href

Reply:
I can't with the existing code. There is no state associated with the action.